index.mjs

```diff
@@ -34,6 +36,21 @@ const concat_text = element => element.children.filter(({ type }) => type === `t
 const read_text_from_descendant = (response_element, ...rest_of_target_names) => concat_text(
 	drill_down_to_children(response_element, ...rest_of_target_names)[0],
 )
+const convert_attribute_elements_to_object = (element) => {
+	const attributes = elements_with_name(element.children, `Attribute`)
+
+	return Object.fromEntries(attributes.map((attribute) => {
+
+		const name = read_text_from_descendant(attribute, `Name`)
+		const text_value = read_text_from_descendant(attribute, `Value`)
+		const int_value = parseInt(text_value)
+
+		return [
+			make_snake_case(name),
+			Number.isNaN(int_value) ? text_value : int_value
```

Comment:
`parseInt` is a bit too lenient to trust, probably – `parseInt('1 or 2 seems fine')` evaluates to `1`.

Maybe 

```suggestion
		const text_value = read_text_from_descendant(attribute, `Value`)
		const is_all_digits = /^\d+$/.test(text_value)

		return [
			make_snake_case(name),
			is_all_digits ? parseInt(text_value, 10) : text_value
```

instead?

It feels bad to have to do duck-typey coercion, but that's probably preferable to just returning strings